packages/rest/.eslintrc.json

```diff
@@ -1,3 +1,11 @@
 {
-	"extends": "../../.eslintrc.json"
+	"extends": "../../.eslintrc.json",
+	"rules": {
+		"n/prefer-global/url": "off",
+		"n/prefer-global/url-search-params": "off",
+		"n/prefer-global/buffer": "off",
```

Comment:
maybe we should leave this one on so it's obvious that we are using node only APIs?